ToolBox/QuantConnect.ToolBox.csproj

```diff
@@ -149,7 +147,7 @@
     <Compile Include="ConsolidatorDataProcessor.cs" />
     <Compile Include="CryptoiqDownloader\CryptoiqBitcoin.cs" />
     <Compile Include="CryptoiqDownloader\CryptoiqDownloader.cs" />
-    <Compile Include="CryptoiqDownloader\Program.cs" />
+	<Compile Include="CryptoiqDownloader\Program.cs" />
```

Comment:
Hey @jaredbroad it's a difference in Xamarin project settings, I will manually update it.